### PR TITLE
Feature/issue 1261 kill wizard

### PR DESCRIPTION
**Pull request by SteveBarnett**
Remove the wizard button and replace it with more helpful new channel, conversation, and router buttons.

### CODE REVIEW

**Comment by SteveBarnett** [comment]
Ready for review, @justinvdm .

File: go/base/static/css/campaigns.less [hunk 1]
```diff
@@ -93,8 +93,8 @@
     text-align:right; // cancel, save draft, next step
 
     // offsets the cancel button a bit.
-    a:first-child {
-      margin-right:20px;
+    #reset {
+      margin-left: 20px;
     }
```
Comment (justinvdm) [inline go/base/static/css/campaigns.less:98]: Wondering if this is going to mess up other pages with elements using `reset` as an id, but sounds unlikely. I know we are already using id's in our css in some places ( /o\ ), but maybe we should try avoiding adding more, and rather add a class to the element or something?

Comment (SteveBarnett) [inline go/base/static/css/campaigns.less:98]: You speak the sense. New commit pushed.

**Comment by justinvdm** [comment]
:+1: once travis is happy.

I think we still need to remove `'go.wizard'` from `INSTALLED_APPS` in `settings.py`, think that could be while the travis build is erroring.

**Comment by SteveBarnett** [comment]
Ready for re-review, please, @justinvdm .

**Comment by justinvdm** [comment]
:+1: once travis is happy

**Pull request merged**